docs/eks-jobs-migration.md

```diff
@@ -0,0 +1,216 @@
+
+The following jobs can be migrated but require some actions to be taken:
+
+- Jobs using external non-GCP resources (e.g. DigitalOcean, vSphere...)
+  - Community ownership of resources is required for jobs running in the new
+    cluster, see the Community Ownership section for more details
+
+The following jobs **MUST NOT** be migrated at this time:
+
+- Jobs using GCP resources (e.g. E2E tests, promotion jobs, etc.)
+- Jobs that are running in trusted clusters (e.g. `test-infra-trusted` and
+  `k8s-infra-prow-build-trusted`)
+
+Jobs that are already running in community-owned clusters (e.g. 
+`k8s-infra-prow-build`) can be migrated as well, but it's not required or
+mandated.
+
+## How To Migrate Jobs?
+
+Fork and check out the [kubernetes/test-infra][k-test-infra] repository,
+then follow the steps below:
+
+[k-test-infra]: https://github.com/kubernetes/test-infra
+
+- Find a job that you want to migrate
+  - You can check out the following ["Prow Results"][prow-results-default] link
+    for recent jobs that are running in the `default` cluster
+  - For explanation reasons, let's say that you picked up a job called
+    `pull-jobset-test-integration-main`
+- Edit the file that `pull-jobset-test-integration-main` is defined in. All
+  jobs are defined in the [kubernetes/test-infra][k-test-infra] repository. You
+  can use GitHub or our [Code Search tool][cs-test-infra] to find the file
+  where this job is defined (search by job name).
+- Look for a `cluster:` key in the job definition. If there isn't one or it's
+  set to `default`, then the job runs in the default cluster. Add (or replace)
+  the following `cluster:` key: `cluster: eks-prow-build-cluster`.
+  - **IMPORTANT: if you see any entries under `label` that says `gce` skip this
+    job and get back to it the next time as this is not ready to be moved yet.**
+  - **IMPORTANT: if you see any entries under `label` that might indicate that
+    a job is using some external resource (e.g. some cloud platform), you need
+    to check if the Community Ownership criteria is satisfied for that resource
+    (see the Community Ownership section for more details)**
+  - **IMPORTANT: jobs running in community-owned clusters must have resource
+    requests and limits specified for cluster-autoscaler to work properly.
+    If that's not the case for your job, please see the next section for some
+    details about determining correct resource requests and limits.**
+  - Here's an [example of a job][example-eks-job] as a reference (pay attention
+    to the `cluster:` key)
+- Save the file, commit the change, create a branch and file a PR
+- Once the PR is merged, follow the guidelines in the Post Migration Period
+  section of this document to ensure that the job remains stable
+
+If you have any trouble, please leave a note in the following
+[GitHub issue][test-infra-gh-issue] or come to
+[#sig-k8s-infra][slack-k8s-infra] or [#sig-testing][slack-sig-testing] Slack
+channel to ask for help.
```

Comment:
This block is duplicated. I'm ok with that, but we could also redirect to the bottom part of this document.